server/src/relay/RelayHttpServer.go

```diff
@@ -4,25 +4,28 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/ethereum/go-ethereum/params"
 	"io/ioutil"
-	"librelay"
-	"librelay/txstore"
 	"log"
 	"math/big"
 	"net/http"
 	"net/url"
 	"os"
 	"path/filepath"
 	"time"
+
+	"../librelay"
```

Comment:
Why do the import paths need to change here? Are you sure that you defined your GOPATH correctly?